src/pandroid/app/src/main/res/layout/fragment_game_drawer.xml

```diff
@@ -58,16 +59,16 @@
                 android:textColor="?colorOnSurfaceVariant"
                 android:textSize="14sp"/>
 
-        </androidx.appcompat.widget.LinearLayoutCompat>
+        </LinearLayout>
 
-    </androidx.appcompat.widget.LinearLayoutCompat>
+    </LinearLayout>
 
     <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_marginTop="10dp">
 
-        <androidx.appcompat.widget.LinearLayoutCompat
+        <LinearLayout
```

Comment:
Use LinearLayoutCompat:
https://developer.android.com/jetpack/androidx?hl=pt-br

Reply:
LinearLayoutCompat is mostly used for android 4.0 or below so it's doesn't matter in this android 7.0 or above